client/me/account-close/main.jsx

```diff
@@ -0,0 +1,143 @@
+	};
+
+	closeConfirmDialog = () => {
+		this.setState( { showConfirmDialog: false } );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const hasAtomicSites = false;
+
+		return (
+			<div className="account-close main" role="main">
+				<HeaderCake onClick={ this.goBack }>
+					<h1>{ translate( 'Close account' ) }</h1>
+				</HeaderCake>
+				<ActionPanel>
+					<ActionPanelTitle className="account-close__heading">
+						{ translate( 'Close account' ) }
+					</ActionPanelTitle>
+					<ActionPanelBody>
+						<ActionPanelFigure>
+							<ActionPanelFigureHeader>
+								{ translate( 'These items will be deleted' ) }
+							</ActionPanelFigureHeader>
+							<ActionPanelFigureList>
+								<ActionPanelFigureListItem>
+									{ translate( 'Personal details' ) }
+								</ActionPanelFigureListItem>
+								<ActionPanelFigureListItem>{ translate( 'Sites' ) }</ActionPanelFigureListItem>
+								<ActionPanelFigureListItem>{ translate( 'Posts' ) }</ActionPanelFigureListItem>
+								<ActionPanelFigureListItem>{ translate( 'Pages' ) }</ActionPanelFigureListItem>
+								<ActionPanelFigureListItem>{ translate( 'Media' ) }</ActionPanelFigureListItem>
+								<ActionPanelFigureListItem>{ translate( 'Domains' ) }</ActionPanelFigureListItem>
+							</ActionPanelFigureList>
+						</ActionPanelFigure>
+						{ ! hasAtomicSites && (
+							<div>
+								<p>
+									{ translate(
+										'Account closure {{strong}}cannot{{/strong}} be undone, ' +
+											'and will remove all sites and content.',
+										{
+											components: {
+												strong: <strong />,
+											},
+										}
+									) }
+								</p>
+								<p>
+									{ translate(
+										"If you're unsure about what account closure means or have any other questions, " +
+											'please {{a}}chat with someone from our support team{{/a}} before proceeding.',
+										{
+											components: {
+												a: <ActionPanelLink href="/help/contact" />,
+											},
+										}
+									) }
+								</p>
+							</div>
+						) }
+						{ hasAtomicSites && (
+							<div>
+								<p>
+									{ translate(
+										"To close your account, you'll need to contact our support team. Account closure cannot be undone " +
+											'and will remove all sites and content.'
+									) }
+								</p>
+								<p>
+									{ translate(
+										"If you're unsure about what account closure means or have any other questions, " +
+											"you'll have a chance to chat with someone from our support team before anything happens."
+									) }
+								</p>
+							</div>
+						) }
+					</ActionPanelBody>
+					<ActionPanelFooter>
+						{ ! hasAtomicSites && (
+							<Button scary onClick={ this.handleDeleteClick }>
+								<Gridicon icon="trash" />
+								{ translate( 'Close account' ) }
+							</Button>
+						) }
+						{ hasAtomicSites && (
+							<Button primary href="/help/contact">
+								{ translate( 'Contact support' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Contact Support' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/me/account-close-confirm-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).